base/generate-ci-config

```diff
@@ -16,14 +16,17 @@ export DEBIAN_FRONTEND=noninteractive
 apt-get update -y -qq > /dev/null
 
 # Generate a list of all kernel versions which have NVIDIA precompiled driver packages available.
-SUPPORTED_KERNELS=$(apt-cache search linux-objects-nvidia-${DRIVER_BRANCH}-server.*${LTS_KERNEL}.*${KERNEL_FLAVOR} | awk '{print $1}' | grep -v "open" | sed -e "s/^.*${LTS_KERNEL}/${LTS_KERNEL}/" | sort -n -t'-' -k2)
+SUPPORTED_KERNELS_LINUX_OBJECTS_LIST=$(apt-cache search linux-objects-nvidia-${DRIVER_BRANCH}-server.*${LTS_KERNEL}.*${KERNEL_FLAVOR} | awk '{print $1}' | grep -v "open" | sed -e "s/^.*${LTS_KERNEL}/${LTS_KERNEL}/" | sort -n -t'-' -k2)
```

Comment:
linux-objects-nvidia is a package that is only installed when installing the proprietary modules. Given than OpenRM is the officially preferred kernel module. I'd propose checking for either of the two packages instead

- `linux-signatures-nvidia.*${LTS_KERNEL}.*${KERNEL_FLAVOR}` (preferred as it's installed in both OpenRM and proprietary)
- `linux-modules-nvidia-${DRIVER_BRANCH}-server.*${LTS_KERNEL}.*${KERNEL_FLAVOR}`